### ISSUE TITLE
isUnique rule for collection on Vuelidate Next

### ISSUE THREAD

**Issue by yuri-kovalev** [vuelidate/vuelidate#1123]
Hey Guys! I researched all issues and don't understand how i can implement my rule `isUnique` for collection. Could you help me? Please :( 

I also looked at https://github.com/vuelidate/vuelidate/issues/781

So, for Vuelidate first version i had this solution
```
validations() {
  return {
    collection: {
      $each: {
        title: {
          isUnique: isUnique(this.collection, 'title')
        }
      }
    }
  };
},
```

and helper 
```
export default function isUnique(group, key) {
  return (value) => {
    const found = group.filter(item => {
      if (key) {
        return item[key] === value;
      }
      return item === value;
    });

    return found.length <= 1;
  };
}
```

How i can migrate my solution for Vuelidate next?

**Comment by yuri-kovalev** [comment]
@dobromir-hristov @shentao hey guys, could you help me please?

**Comment by dobromir-hristov** [comment]
Should work like how you wrote it. 

Create a simple reproducible example and we can try to help you.

**Comment by yuri-kovalev** [comment]
> Should work like how you wrote it.
> 
> Create a simple reproducible example and we can try to help you.

@dobromir-hristov but you removed `$each`, right?

**Comment by dobromir-hristov** [comment]
Yes, but you could use the eachHelper, we just really don't recommend it haha.

For collections it's better to use  another component for each object.

**Comment by yuri-kovalev** [comment]
This is very inconvenient for me, use another component for each object :(

**Comment by dobromir-hristov** [comment]
Create a simple repro in codesandbox and we can try to help
